Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -1415,6 +1416,13 @@ public static void SetPlayerName(string name, Action<PlayerNameResponse> onCompl
                 return;
             }
 
+            if (name.Contains(PlayerPrefs.GetString("LootLockerGuestPlayerID"), StringComparison.OrdinalIgnoreCase) && CurrentPlatform.Get() == Platforms.Guest)
```

Comment:
Invert this if statement. The string comparison is relatively expensive and the platform comparison is cheap. So we want to do the latter one first.